tasks/linter.py

```diff
@@ -351,3 +358,27 @@ def is_get_parameter_call(file):
                         return SSMParameterCall(file, nb, with_wrapper=True, with_env_var=False)
         except UnicodeDecodeError:
             pass
+
+
+@task
+def gitlab_ci(_, test="all", custom_context=None):
+    """
+    Lint Gitlab CI files in the datadog-agent repository.
+    """
+    all_contexts = []
+    if custom_context:
+        all_contexts = load_context(custom_context)
+    else:
+        all_contexts = get_preset_contexts(test)
+    print(f"We will tests {len(all_contexts)} contexts.")
+    for context in all_contexts:
+        print("Test gitlab configuration with context: ", context)
+        config = generate_gitlab_full_configuration(".gitlab-ci.yml", dict(context))
+        gitlab = Gitlab(api_token=get_gitlab_token())
+        res = gitlab.lint(config)
+        print(f"Config is valid: {res['valid']}")
+        if len(res["warnings"]) > 0:
+            print(f"Warnings: {res['warnings']}")
+        if not res["valid"]:
+            print(f"Errors: {res['errors']}")
```

Comment:
Should this be printed in red / on stderr / using a logger ? (same for warnings)

Reply:
Used the current solution (`color_message`). Will involve a logger later